### PR TITLE
07 support

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

File: src/WinRPM.jl [hunk 1]
```diff
@@ -451,21 +452,23 @@ const exe7z = iswindows() ? joinpath(BINDIR, "7z.exe") : "7z"
...
     cache = getcachedir(source)
-    path2 = joinpath(cache,escape(path))
+    path2 = joinpath(cache, escape(path))
     open(path2, "w") do f
         write(f, data[1])
     end
-    info("Extracting: ", name)
+    @info("Extracting: ", name)
     cpio = splitext(path2)[1]*".cpio"
+    cpio = replace(cpio, "noarch%2F", "")
```
Comment (SimonDanisch) [inline src/WinRPM.jl]: I'm not really sure why this was suddenly needed... Are these related to changes in the binary repositories?

Comment (tkelman) [reply src/WinRPM.jl]: No. See #141. Did you test this on 0.6, actually installing anything from this branch?

Comment (SimonDanisch) [reply src/WinRPM.jl]: Oh I guess I should just close this in favor of #141?

Comment (SimonDanisch) [reply src/WinRPM.jl]: Well, better rebase this one against #141 I guess

**Comment by musm** [comment]
rebase?
